Source/com/drew/metadata/mp4/Mp4BoxHandler.java

```diff
@@ -90,8 +90,6 @@ public boolean shouldAcceptContainer(@NotNull Box box)
             } else if (box.type.equals(Mp4BoxTypes.BOX_USER_DEFINED)) {
                 Mp4UuidBoxHandler userBoxHandler = new Mp4UuidBoxHandler(metadata);
                 userBoxHandler.processBox(box, payload, context);
-            } else if (box.type.equals(Mp4BoxTypes.BOX_USER_DATA)) {
-                processUserData(box, reader, payload.length);
```

Comment:
"udta" is a container. We can move this underlying logic to the ItemListBox (where it belongs).